packages/contracts/test/core/dao/dao.ts

```diff
@@ -239,22 +239,56 @@ describe('DAO', function () {
         .withArgs([0, 1, 0]);
     });
 
-    it('reverts if trying to upgrade to the same version', async () => {
+    it('initializes `_reentrancyStatus` for versions < 1.3.0', async () => {
+      // Create an unitialized DAO.
       const uninitializedDao = await deployWithProxy<DAO>(DAO);
 
-      await expect(uninitializedDao.initializeFrom([1, 3, 0], EMPTY_DATA))
-        .to.be.revertedWithCustomError(
-          dao,
-          'ProtocolVersionUpgradeNotSupported'
-        )
-        .withArgs([1, 3, 0]);
+      // Expect the contract to be uninitialized  with `_initialized = 0` and `_reentrancyStatus = 0`.
+      expect(
+        ethers.BigNumber.from(
+          await ethers.provider.getStorageAt(
+            uninitializedDao.address,
+            OZ_INITIALIZED_SLOT_POSITION
+          )
+        ).toNumber()
+      ).to.equal(0);
```

Comment:
Shouldn't this be `1` after deployed and initialized? Or does the first initialization start at 0?

Reply:
In this unit tests, I deploy an uninitialized DAO for testing purposes, so it is expected to be 0.

In the integration tests in folder `packages/contracts/test/upgrade/dao.ts`, we do the actual checks for the upgrade.

Here, I am just testing the current implementation and I don't want to mix other implementations in.